packages/jest-config/src/normalize.js

```diff
@@ -154,23 +156,28 @@ const normalizeUnmockedModulePathPatterns = (
   return config[key].map(pattern =>
     utils.replacePathSepForRegex(
       pattern.replace(/<rootDir>/g, config.rootDir),
-    )
-  );
+    ));
 };
 
 const normalizePreprocessor = (config: InitialConfig) => {
   /* eslint-disable max-len */
   if (config.scriptPreprocessor && config.transform) {
     throw createConfigError(
-`  Options: ${chalk.bold('scriptPreprocessor')} and ${chalk.bold('transform')} cannot be used together.
-  Please change your configuration to only use ${chalk.bold('transform')}.`
+      `  Options: ${chalk.bold('scriptPreprocessor')} and ${chalk.bold(
+        'transform',
+      )} cannot be used together.
```

Comment:
This is a nice tricky way to avoid extra whitespace! I'm not seeing myself writing template literals like that, but it's perfectly fine when a 3rd party tool does it for me.